src/bun.js/javascript.zig

```diff
@@ -1557,6 +1567,11 @@ pub const VirtualMachine = struct {
             this.pending_internal_promise = promise;
         }
 
+        if (this.auto_inspect) {
+            this.auto_inspect = false;
+            _ = this.global.startRemoteInspector("0.0.0.0", 9229);
```

Comment:
```suggestion
            _ = this.global.startRemoteInspector("127.0.0.1", 9229);
```

Let's prevent accidental RCE at the coffee shop.